elastic/kibana
# [ILM] Fix frozen phase min_age (de)serialization

This PR fixes an issue where the `min_age` for frozen phase was neither deserialized nor serialized.

### How to test

* Create a new policy
* Enable the frozen phase and define a custom min age
* The policy should save correctly and you should be redirected to the list of policies
* Now edit the same policy
* The frozen phase should be enabled and the custom min age should correctly be set

Fixes https://github.com/elastic/kibana/issues/96392
